rspace/src/test/scala/coop/rchain/rspace/StorageActionsTests.scala

```diff
@@ -884,7 +887,8 @@ class InMemoryStoreStorageActionsTests extends StorageActionsTests with JoinOper
 class LMDBStoreActionsTests
     extends StorageActionsTests
     with JoinOperationsTests
-    with BeforeAndAfterAll {
+    with BeforeAndAfterAll
+    with IStoreTests {
```

Comment:
What if we composed the `IStoreTests` separately this separately?  It would make them easier to run on their own.

Reply:
Ok, will do that